components/test_raftstore/src/common-test.toml

```diff
@@ -41,7 +41,6 @@ capacity = "64MB"
 [metric]
 
 [raftstore]
-sync-log = false
```

Comment:
Should be in next PR.

Reply:
The `sync-log` has already been removed in master.